debian/control

```diff
@@ -0,0 +1,30 @@
+Source: quicktile
+Maintainer: Stéphane Gourichon <stephane_dpkg@gourichon.org>
+Section: misc
+Priority: optional
+Build-Depends: debhelper (>= 9),
+               build-essential,
+               devscripts,
+               dh-python,
+               fakeroot,
+               python3,
+               python3-pip,
+               python3-setuptools,
+               python3-gi,
+               python3-xlib,
+               python3-dbus,
+               gir1.2-gtk-3.0,
```

Comment:
`gir1.2-glib-2.0` became an additional dependency since this was written.

Reply:
Ok. Build succeeds on Debian (minideb) anyway without mentioning or installing `gir1.2-glib-2.0` because it is pulled automatically as dependency of others. Adding it as explicit dependency.